config/routes.rb

```diff
@@ -9,6 +9,7 @@
       resources :sessions, only: %i[index create destroy]
       resources :users, only: [:create]
       resources :rooms, only: %i[show index], param: :friendly_id
+      post '/room/:friendly_id/start', to: 'rooms#start', as: :start_session
```

Comment:
There's a cleaner way to do this - https://guides.rubyonrails.org/routing.html#adding-more-restful-actions

Reply:
Alright, I'll check it.

Reply:
Thank you!

Reply:
DONE